easybuild/easyconfigs/f/FlexiBLAS/FlexiBLAS-3.3.1-NVHPC-23.1-CUDA-12.0.0.eb

```
@@ -0,0 +1,73 @@
+easyblock = 'Bundle'
+
+name = 'FlexiBLAS'
+version = '3.3.1'
+
+homepage = 'https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release'
+description = """FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementation
+used by a program without recompiling or relinking it."""
+
+toolchain = {'name': 'NVHPC', 'version': '23.1-CUDA-12.0.0'}
+local_extra_flags = "-D__ELF__"
+toolchainopts = {'pic': True, 'extra_cflags': local_extra_flags, 'extra_fflags': local_extra_flags}
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('Python', '3.10.8', '-bare'),  # required for running the tests
+]
+
+dependencies = [
+    ('OpenBLAS', '0.3.24'),
+]
+
+# note: first listed backend will be used as default by FlexiBLAS,
+# unless otherwise specified via easyconfig parameter flexiblas_default
+local_backends = ['OpenBLAS']
+
+# imkl supplies its backend via the imkl module, not as a dependency
+if ARCH == 'x86_64':
+    local_backends.append('imkl')
+
+default_component_specs = {'start_dir': '%(namelower)s-%(version)s'}
+sanity_check_all_components = True
+
+# Also build and install LAPACKE, which FlexiBLAS does not support yet
+components = [
+    (name, version, {
+        'source_urls': ['https://github.com/mpimd-csc/flexiblas/releases/download/v%(version)s/'],
+        'sources': [SOURCELOWER_TAR_GZ],
+        'patches': [
+            'FlexiBLAS-3.0.4_fix-imkl.patch',
+            'FlexiBLAS-3.0.4_fix-wrapper_blas_intel.patch',
+            'FlexiBLAS-3.0.4_remove-semicolon.patch',
+            'FlexiBLAS-3.2.0_remove-intel-suffix.patch',
+        ],
+        'checksums': [
+            'bbeebf5e5a006924558fec43f49affbe1aaa4cbacfc472a9ff6066ffda142e18', # flexiblas-3.3.1.tar.gz
+            'a078e46ad126574cc42314abd5b40461cdc6e950af79d105dbf92875c254c87c',  # FlexiBLAS-3.0.4_fix-imkl.patch
```

```suggestion
            'a078e46ad126574cc42314abd5b40461cdc6e950af79d105dbf92875c254c87c', # FlexiBLAS-3.0.4_fix-imkl.patch
```

I think this is what the style check/linter complains about.